### ISSUE TITLE
Option to group tabs in panel by container

### ISSUE THREAD

**Issue by jaybhanushali3195** [mbnuqw/sidebery#1414]
## Description
**Current behavior**
When I open various tabs in a container, it just shows colors
![image](https://github.com/mbnuqw/sidebery/assets/36804441/32ece699-f133-4f6c-9e76-18e2d5e193fa)

**Expected behavior** 
I am using **container tabs extension** to get this functionality
![image](https://github.com/mbnuqw/sidebery/assets/36804441/76bd69c9-0665-469a-9550-3c9626a56e64)

Can this be done in sideberry already ?

**Comment by jaybhanushali3195** [comment]
Similar requirement as this
https://github.com/mbnuqw/sidebery/issues/1220

**Comment by mbnuqw** [comment]
I'll update the title to reflect the expected behavior: Option to group tabs in panel by container.

**Comment by Marin-Kitagawa** [comment]
Yeah, I was about to create a new feature request for this as well. Could you please implement this? It'd be really helpful.

**Comment by k5123** [comment]
Seconded. Kind of an obvious thing to do, tbh.
